web/oauth.mjs

```diff
@@ -0,0 +1,86 @@
+import * as oauth from 'https://cdn.jsdelivr.net/npm/oauth4webapi@3.8.2/+esm';
+
+export async function fetchConfig() {
```

Comment:
As discussed above, this was the "best worst" solution I could come up with for reading the config from the plain html pages. Reading dart defines would involve booting up Flutter here.